cds_dojson/schemas/records/definitions-v1.0.0.json

```diff
@@ -3,52 +3,6 @@
   "$schema": "http://json-schema.org/draft-04/schema#",
   "type": "object",
   "definitions": {
-    "creator": {
```

Comment:
Do not remove the creator from here as we might need it for other type of record.